apps/extension/src/state/tx-approval.ts

```diff
@@ -1,13 +1,12 @@
 import { AllSlices, SliceCreator } from './index';
 import { AuthorizeRequest } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/custody/v1alpha1/custody_pb';
-import { Jsonified } from '@penumbra-zone/types';
 import { MessageResponder } from '@penumbra-zone/types/src/internal-msg/shared';
 import { TransactionView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1alpha1/transaction_pb';
 import { TxApproval } from '@penumbra-zone/types/src/internal-msg/tx-approval';
 
 export interface TxApprovalSlice {
-  authorizeRequest?: Jsonified<AuthorizeRequest>;
-  transactionViewFromPlan?: Jsonified<TransactionView>;
+  authorizeRequest?: AuthorizeRequest;
+  transactionViewFromPlan?: TransactionView;
```

Comment:
not sure if zustand will tolerate this. other option is serialize to string because zustand doesn't tolerate `JsonValue`

Reply:
it doesn't like it, storing strings now